### ISSUE TITLE
Log items appear to be using different timezones

### ISSUE THREAD

**Issue by nordam** [OpenDrift/opendrift#1416]
Here is an extract from a log:

```
17:09:30 INFO    opendrift.models.basemodel.environment:248: 	turbulent_kinetic_energy: 0.000000
17:09:30 INFO    opendrift.models.basemodel.environment:248: 	turbulent_generic_length_scale: 0.000000
17:09:30 INFO    opendrift.models.basemodel.environment:248: 	upward_sea_water_velocity: 0.000000
15:09:32 INFO    opendrift.models.basemodel:936: Using existing reader for land_binary_mask
15:09:32 INFO    opendrift.models.basemodel:947: All points are in ocean
```

Local time as I was running this was 17:09, so it appears to switch to UTC or something?

**Comment by knutfrode** [comment]
Yes, this is very strange.

I have not seen this before, and am not able to reproduce.
But if you can provide a reproducible example, I can try to fix this.
I assume we want local system time in the logs, and not necessarily UTC.

**Comment by gauteh** [comment]
Maybe it is related to asctime (https://github.com/OpenDrift/opendrift/blob/master/opendrift/models/basemodel/__init__.py#L269) using UTC?

**Comment by knutfrode** [comment]
Yes but I cannot see why it should suddenly change in the middle of a simulation?
It should be the same logger, although within the OpenDrift codemass the logger is sometimes inconsistently retrieved as either
`logger = logging.getLogger(__name__)`  or
`logger = logging.getLogger('opendrift')`

**Comment by gauteh** [comment]
True. It could be that the netcdf-library or some C library is changing the locale, libnetcdf does a lot of weird things.

**Comment by knutfrode** [comment]
Yes, I agree to blame this one on others :-)

**Comment by gauteh** [comment]
Great!

**Comment by knutfrode** [comment]
GitHub need a resolve button for issues with "Close as blamed on others".

**Comment by gauteh** [comment]
Is this also fixed with the recent logging fixes?

**Comment by knutfrode** [comment]
Hopefully, but copernicusmarine is doing many things with the logging (but so is OpenDrift).
So closing this for now.
